schema/dataKeys.json

```diff
@@ -0,0 +1,383 @@
+        },
+        "product_stream": {
+          "type": "string",
+          "description": "The product stream e.g. RHEL-tp1"
+        },
+        "cpe": {
+          "type": "string",
+          "description": "The product CPE ID e.g. cpe:/a:example:openstack:el8"
+        },
+        "type": {
+          "type": "string",
+          "enum": ["RHEA", "RHBA", "RHSA"],
+          "description": "Type advisory type e.g. RHSA"
+        },
+        "synopsis": {
+          "type": "string",
+          "description": "The advisory synopsis e.g. my advisory synopsis"
+        },
+        "topic": {
+          "type": "string",
+          "description": "The advisory topic e.g. my advisory topic"
+        },
+        "description": {
+          "type": "string",
+          "description": "The advisory description e.g. This advisory is for security fixes to my product"
+        },
+        "solution": {
+          "type": "string",
+          "description": "The advisory solution e.g. For details on how to apply this update, see docs.com"
+        },
+        "references": {
+          "type": "array",
+          "description": "The advisory references e.g. [ 'https://access.redhat.com/security/updates/classification', 'https://docs.redhat.com/some/example/release-notes' ] ",
+          "items": {
+            "type": "string"
+          }
+        },
+        "issues": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "fixed": {
+              "type": "array",
+              "items": {
+                "additionalProperties": false,
+                "type": "object",
+                "properties": {
+                  "id": {
+                    "type": "string",
+                    "description": "The ID of the fixed issue e.g. RHOSP-3414"
+                  },
+                  "source": {
+                    "type": "string",
+                    "description": "The URL of the where the issue is tracked e.g. bugzilla.example.com"
+                  }
+                }
+              }
+            }
+          }
+        },
+        "content": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "images": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "additionalProperties": false,
+                "properties": {
+                  "containerImage": {
+                    "type": "string",
+                    "description": "The container image e.g. quay.io/example/openstack@sha256:example"
+                  },
+                  "repository": {
+                    "type": "string",
+                    "description": "The repository e.g. rhosp16-rhel8/openstack"
```

Comment:
Can you change the example repository to include registry.redhat.io/ before rhosp16-rhel8?

Reply:
fixed thank you